training/trillium/Mixtral-8x7B-Pytorch/XPK/benchmark.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+source env.sh
```

Comment:
I think this needs to be called before XPK cluster creation step.

Reply:
Done